src/analyses/ai.cpp

```diff
@@ -366,6 +368,7 @@ bool ai_baset::visit_edge_function_call(
     p_call,
     calling_function_id,
     l_return,
+    nullptr, // Not needed as we are skipping the function call
```

Comment:
This and other `nullptr` args are pretty opaque -- how about `#define CALLER_UNSPECIFIED nullptr`?

Reply:
I don't really agree with using a define for this. How about `constexpr whatever* UnspecifiedCaller = nullptr;`?

Reply:
Added to ai_historyt as a member variable as @hannes-steffenhagen-diffblue suggests.

Reply:
( Unfortunately `constexpr` doesn't actually work because `ai_history_baset` has a virtual destructor but a `static const` variable does. )